Template/Tag/MatomoTag.web.js

```diff
@@ -115,7 +120,7 @@
 
                 var setCustomDimensionIndexes = [];
                 indexesToRemove = [];
-                var _paq = JSON.parse(JSON.stringify(initialPaq));
+                var localPaq = JSON.parse(JSON.stringify(initialPaq));
```

Comment:
Just for my understanding was this variable name conflicting with tests ?

Reply:
That's the only thing that I can think of, cause my change around `window._paq` didn't make the build pass.

Reply:
okay